.github/workflows/release.yml

```diff
@@ -89,21 +89,9 @@ jobs:
         if: ${{steps.semantic-release.outputs.TAG_NAME != ''}}
         run: node .github/scripts/update-release.js
         env:
-          REPO_ACCESS_TOKEN: ${{ github.token }}
           TAG_NAME: ${{steps.semantic-release.outputs.TAG_NAME}}
           RELEASE_BODY: ${{github.event.inputs.change-log}}
           REPO_NAME: twilio/twilio-cli
-  delete-draft-release:
-    if: ${{github.event.inputs.draft-release == 'true' && needs.release.outputs.draft-tag-name != ''}}
-    runs-on: ubuntu-latest
-    needs: [release]
-    steps:
-    - name: Delete draft release
-      if: ${{github.event.inputs.draft-release == 'true'}}
-      run: node .github/scripts/trigger-workflow.js
-      env:
-          WORKFLOW_NAME: '.github/workflows/delete-draft-tag-release.yml'
```

Comment:
Where are we triggering the cleanup job?

Reply:
Restored it, was testing trigger deletion for all steps together.